toolchains/make_package/vanilla/makepkg.conf

```diff
@@ -0,0 +1,141 @@
+#########################################################################
```

Comment:
This file hasn't changed, it's just been moved. Git commit failed to recognise that it was moved because a similar file (android/makepkg.conf) also was moved.